src/ipelib/ipeplatform.cpp

```diff
@@ -400,16 +401,18 @@ void ipeDebugBuffer(Buffer data, int maxsize) {
 // --------------------------------------------------------------------
 
 //! Returns current working directory.
-/*! Returns empty string if something fails. */
 String Platform::currentDirectory() {
-#ifdef WIN32
-    wchar_t * buffer = _wgetcwd(nullptr, 0);
-    return String(buffer);
-#else
-    char buffer[1024];
-    if (getcwd(buffer, 1024) != buffer) return String();
-    return String(buffer);
-#endif
+    return std::filesystem::current_path().generic_string();
```

Comment:
`generic_u8string`

I'm tempted to say we get rid of `Platform::currentDirectory` and just call `std::filesystem` directly in the code, but that's something I can look at later.